CardanoSharp.Wallet.Test/Bech32Tests.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace CardanoSharp.Wallet.Test
+{
+    public class Bech32Tests
```

Comment:
just playing around for now, could include valid / invalid examples in the future